core/tests/performance_framework/selenium_driver.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This file contains an implementation which interacts with Selenium and
+Browsermob-proxy to fetch performance data.
+"""
+
+import urlparse
+import os
+
+# We need to import browsermob-proxy and selenium. To do so, we need to add
+# add their directories to path.
+CURR_DIR = os.path.abspath(os.getcwd())
+OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
+
+DIRS_TO_ADD_TO_SYS_PATH = [
+    os.path.join(OPPIA_TOOLS_DIR, 'browsermob-proxy-0.7.1'),
```

Comment:
I think I'll move it to the script that runs performance tests. Then, it should be easy!